definition-map.js

```diff
@@ -75,6 +81,16 @@ const QUERIES = {
             includeCorrectAnswers: "Flag to include questions the user previously got correct.",
             quizzes: "The IDs of the Quizzes user selects."
         }
+    },
+    UserContentItems: {
+        metadescription: "Returns an array of content that the current user logged in has access to.",
+        args: {}
+    },
+    Pages: {
+        metadescription: "Returns an array of Page objects that are found within courses, such as quiz pages, text pages, video pages, and more.",
+        args: {
+            identifiers: "An arrary of page IDs"
```

Comment:
These can also be slugs if the topic was created via Helium instead of authored in TI. Obviously not quite applicable at the moment, but we may want a ticket or something to update this once that functionality is out.